Studio-Lovelies/GG-JointJustice-Unity
# Upgrades to editor version 2022.3.4f1

## Summary

Upgrades the Unity version to the current LTS release 2022.3.4f1.

## Testing instructions

Run the tests, play the game, do whatever you did before, and make sure it still works!

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:
